spec/src/main/asciidoc/ServerPages.adoc

```diff
@@ -3247,11 +3183,11 @@ expressions:
 * The `name` attribute of `jsp:element`
 (StandardActions.html#37936[]).
```

Comment:
This is one to take care of later.

Reply:
We can give the document another pass. I left some of the links as I wasn't quite sure at a glance what needed to be done and figured we could give deeper investigation after this PR.  Thanks for flagging them!

Reply:
Yeah, I'm good with forward progress now.
Final touches later.